armasec/armasec.py

```diff
@@ -30,6 +31,8 @@ def __init__(
 
         Args:
             domain:           The OIDC domain where resources are loaded
+            use_https:        If falsey, use ``http`` when pulling openid config from the OIDC
```

Comment:
typo

Reply:
Where? If you mean "falsey", it's a term of art meaning a value that resolved to False when used in a boolean expression.

`None`, `''`, and `[]` are all "falsey" values.

Reply:
ohhh, nice. I didn't know, thank you